### DISCUSSION TITLE
Δήλωση πρόθεσης της προαιρετικής παρουσίασης

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#132] [Announcements]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης  ερευνητικού άρθρου στέλνοντας το τελικό αρχείο παρουσίασης

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη ΟΣΣ

**Comment by Maurphal** [comment]
Δηλώνω την πρόθεση μου να παρουσιασω άρθρο την Κυριακή!
[Μέρος Γ - Παρουσίαση επιστημονικού άρθρου AR-VR-XR.pptx](https://github.com/sdy60/syllabus/files/14445440/-.AR-VR-XR.pptx)
Virtual Reality (2019) 23:101–118
https://doi.org/10.1007/s10055-018-0364-1

**Comment by std159312** [comment]
Δηλώνω και εγώ την πρόθεση μου για την παρουσίαση άρθρου σχετικά με το AR την Κυριακή 3/3.

[Παρουσίαση επιστημονικού άρθρου AR.pptx](https://github.com/sdy60/syllabus/files/14470271/AR.pptx)

https://doi.org/10.1145/3290605.3300774

**Comment by IoannisTheodoropoulos** [comment]
Καλησπέρα σας, θα ήθελα να δηλώσω κι εγώ την πρόθεσή μου για την παρουσίαση την Κυριακή 3/3, ευχαριστώ!
[VR Interaction.pptx](https://github.com/sdy60/syllabus/files/14470211/VR.Interaction.pptx)

https://link.springer.com/article/10.1007/s10055-017-0318-z

**Comment by Asinapis13** [comment]
Καλημέρα σας,

θα ήθελα να δηλώσω την πρόθεση μου για παρουσίαση της διαφάνειας στις 10-3-2024

[ΣΔΥ 60 4η ΓΕ παρουσίαση.pptx](https://github.com/sdy60/syllabus/files/14535474/60.4.pptx)

**Reply by Asinapis13** [reply]
Επισύναψα και το doi του άρθρου

https://doi.org/10.1109/TLT.2014.2307297

**Comment by Konssven** [comment]
Καλησπέρα

Θα ήθελα και εγώ να δηλώσω την πρόθεση μου για παρουσίαση την 10/03/2024
[SDY60_4ERG_SVENTZOYRIS_KONSTANTINOS.pdf](https://github.com/sdy60/syllabus/files/14547815/SDY60_4ERG_SVENTZOYRIS_KONSTANTINOS.pdf)

https://doi.org/10.1145/3172944.3172977

**Comment by iparthenios** [comment]
Δηλώνω την πρόθεσή μου να παρουσιάσω σήμερα Κυριακή 10/3/24 σχετικά με το άρθρο:
MagicFace: Stepping into Character through an Augmented Reality Mirror
https://dl.acm.org/doi/abs/10.1145/3025453.3025722

**Reply by iparthenios** [reply]
[MagicFace_ Stepping into Character through an Augmented Reality Mirror.pdf](https://github.com/sdy60/syllabus/files/14549594/MagicFace_.Stepping.into.Character.through.an.Augmented.Reality.Mirror.pdf)
